packages/amplify-app/src/call-amplify.js

```diff
@@ -45,7 +45,7 @@ const callNodeAmplify = async (args, opts) => {
 
 const callPkgAmplify = isWin ? callPkgAmplifyWin : callPkgAmplifyNix;
 
-const callAmplify = !!process.pkg ? callPkgAmplify : callNodeAmplify;
+const callAmplify = process.pkg ? callPkgAmplify : callNodeAmplify;
```

Comment:
be careful about removing `!!` . Last time we removed it it backfired as it removed coercion...

Reply:
If you look up history of this line
it went from
`export const isCI = (): boolean => !!(process.env.CI && process.env.CIRCLECI);` (it was in different file at that time)
through
`export const isCI = (): boolean => process.env.CI && process.env.CIRCLECI;` (causing chaos)
to current form
`export const isCI = (): boolean => JSON.parse(process.env.CI || 'false') && JSON.parse(process.env.CIRCLECI || 'false');`